README.md

```diff
@@ -145,7 +145,8 @@ refer often to the following Conjur configuration variables:
 - `host_factory_token`: The Conjur host factory token, provided as a string or using the
   [Puppet file resource type](https://puppet.com/docs/puppet/latest/types/file.html).
 - `cert_file`: The file path for the PEM-encoded x509 CA certificate chain for the DAP
-  instance you are connecting to. This configuration parameter overrides `ssl_certificate`.
+  instance you are connecting to. This file is read from the **Puppet server**. This
```

Comment:
This could just be my general ignorance about Puppet, but in this instance, does this mean the master or the agent node?

Also, a few more questions:
1. Do the user have to worry about file permissions?
2. If the file isn't found or the process doesn't have permission to read it, how do they debug this?
3. If the file format is incorrect, how might they debug that issue?

Thanks!

Reply:
@alexkalish Oops - I didn't see you had a drive-by comment! 

> Do the user have to worry about file permissions?

Generally no since this is on the server or in windows registry (so it's covered with our current docs

> If the file isn't found or the process doesn't have permission to read it, how do they debug this?

I am not sure but I believe that it would error out with "unable to read file" kind of error

> If the file format is incorrect, how might they debug that issue?

You would get the same "error" (I use this term _very_ loosely) that you would get with a bad `ssl_certificate` value. There is a [bug](https://github.com/cyberark/conjur-puppet/issues/115) filed for that to be improved though.

Reply:
> in this instance, does this mean the master or the agent node?

Puppet master. I could be wrong but I think the specific language here may be to move away from "master" as a term in docs. Since you only have clients (agents) and a server (master), I think it would be mostly obvious to a Puppet operator what we mean there.

Reply:
Thanks for your replies.  I think it's important that we minimally verify that error messages from the above issues have understandable and ideally actionable messages that appear in obvious places.  It looks like the linked bug speaks to that need.  Can that issue also cover the scenarios I mentioned above?  Or do we need a new one?